Scripts/developer_scripts/create_new_release

```diff
@@ -290,10 +302,16 @@ fi
 # Build public release version
 if [ -n "$DO_PUBLIC" ]; then
     echo "Making the public version of the tarball"
-    public_release_number="1${major_nr}${minor_nr}${bugfix_nr}1000"
+    if [ -n "$BETA" ]; then
+      public_release_number="1${major_nr}${minor_nr}${bugfix_nr}09${BETA}0"
+    else
+      public_release_number="1${major_nr}${minor_nr}${bugfix_nr}1000"
+    fi
     public_release_version="${MAJOR_NUMBER}.${MINOR_NUMBER}${BUGFIX_STRING}"
     if [ -r "${NUMBERS_DIR}/public_release_name" ]; then
 	public_release_name=`cat "${NUMBERS_DIR}/public_release_name"`
+    elif [ -n "$BETA" ]; then
+	public_release_name="CGAL-${public_release_version}-beta${$BETA}"
```

Comment:
You should inverse the logic: if `$BETA` is set, do not look in the file `public_release_name`.